src/vl-mapactions-custom-map.js

```diff
@@ -28,9 +28,23 @@ export class VlCustomMap extends VlMapWithActions {
       }),
     ].concat(options.controls || []);
 
+    const projection = options.projection;
+    const view = new View({
+      extent: projection.getExtent(),
+      projection: projection,
+      maxZoom: 16,
+      minZoom: 2,
+      center: [140860.69299028325, 190532.7165957574],
+      zoom: 2,
+      ...options.view,
```

Comment:
Wat wordt er nog voorzien via options.view? Ik zou verwachten dat je via options.view een volledige `View` meegeeft, maar dat blijkt niet uit deze code.

Reply:
Blijkbaar wordt dit in de `vl-map` gebruikt om de extent en projecten te overschrijven? Waarom wordt de extent en projection niet rechtstreeks geconfigureerd?

Reply:
@coemans dit wordt gebruikt om de view te kunnen overschrijven. Default is de view gebasseerd op de projectie, zoals het in het begin was. Maar nu kan bij constructie van de custom map gekozen worden om de view te overschrijven.